deploy/helm/magda/charts/sleuther-broken-link/templates/cronjob.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: sleuther-broken-link-recrawl
+spec:
+  concurrencyPolicy: Allow
+  schedule: "0 0 14,28 * *"
+  failedJobsHistoryLimit: 1
+  successfulJobsHistoryLimit: 1
+  startingDeadlineSeconds: 120
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          name: sleuther-broken-link-recrawl
+          labels:
+            cron: sleuther-broken-link-recrawl
+        spec:
+          containers:
+          - image: "dtzar/helm-kubectl"
```

Comment:
Could this just be `alpine` ?

Reply:
Yes. `alpine` should work as well.
Will update it and let you know~